src/components/flyout/flyout.tsx

```diff
@@ -151,6 +151,12 @@ interface _EuiFlyoutProps {
    */
   pushMinBreakpoint?: EuiBreakpointSize | number;
   style?: CSSProperties;
+  /**
+   * Object of props passed to EuiFocusTrap.
+   * Use `shards` to allow clicks on the provided external elements.
+   * Use `closeOnMouseup` to delay the close callback to prevent conflicts with toggle buttons.
```

Comment:
```suggestion
   * `shards` specifies an array of elements that will be considered part of the flyout, preventing the flyout from being closed when clicked.
   * `closeOnMouseup` will delay the close callback, allowing time for external toggle buttons to handle close behavior.
```

I really liked the copy in your original docs and thought it was slightly more understandable! Up to you which you prefer though